src/dataconnect/fileUtils.ts

```diff
@@ -106,3 +106,29 @@ export async function pickService(
   }
   return serviceInfo;
 }
+
+// case insensitive exact match indicators for supported app platforms
+const WEB_INDICATORS = ["package.json", "package-lock.json", "node_modules"];
+const IOS_INDICATORS = ["info.plist", "project.pbxproj", "podfile", "package.swift"];
```

Comment:
project.pbxproj is a file internal to the .xcodeproj folder (macOS package). We shouldn't create anything as a peer to this since this folder is internal to xcode and managed by xcode. I'd suggest not using this as an ios indicator. Instead just rely on xcodeproj and xcworkspace.

Reply:
done